date.go

```diff
@@ -138,7 +139,16 @@ func (dst *Date) DecodeText(ci *ConnInfo, src []byte) error {
 	case "-infinity":
 		*dst = Date{Status: Present, InfinityModifier: -Infinity}
 	default:
-		t, err := time.ParseInLocation("2006-01-02", sbuf, time.UTC)
+		if sbuf[len(sbuf)-2:] == "BC" {
```

Comment:
if len(sbuf) > 2 && ...

Reply:
or actually I would prefer strings.HasPrefix(" BC") instead

Reply:
yes that is nice :)